domain/api/atrium-domain-api-common/src/main/kotlin/ch/tutteli/atrium/domain/creating/IterableAssertions.kt

```diff
@@ -37,4 +39,6 @@ interface IterableAssertions {
     fun <E : Any> hasNext(expect: Expect<Iterable<E>>): Assertion
 
     fun <E : Any> hasNotNext(expect: Expect<Iterable<E>>): Assertion
+
+    fun <E:Comparable<E>, T : Iterable<E>> min(assertionContainer: Expect<T>): ExtractedFeaturePostStep<T, E?>
```

Comment:
See comment below why the return type change
```suggestion
    fun <E : Comparable<E>, T : Iterable<E>> min(assertionContainer: Expect<T>): ExtractedFeaturePostStep<T, E>
```